packages/protocol/contracts/L1/libs/LibUtils.sol

```diff
@@ -124,4 +125,24 @@ library LibUtils {
             lastVerifiedBlockId: b.lastVerifiedBlockId
         });
     }
+
+    function getInstance(
```

Comment:
Now the "instance" is only an internal concept to PSE-ZKEVM, It should be removed from libs/*.sol.

Reply:
I suggest to move it to PseZkVerifer.sol

Reply:
Ok, i needed for testing. Anyways i put this function in the `TestBase` too then.